ores/wsgi/routes/scores.py

```diff
@@ -7,32 +7,32 @@
 from ..util import ParamError, read_bar_split_param
 
 
-def configure(config, bp, scorers):
+def configure(config, bp, score_processor):
 
     # /scores/
     @bp.route("/scores/", methods=["GET"])
     def scores():
-        wikis = [wiki for wiki in scorers]
+        contexts = [context for context in score_processor]
 
-        wikis.sort()
+        contexts.sort()
 
-        return jsonify({'wikis': wikis})
+        return jsonify({'contexts': contexts})
 
     # /scores/enwiki/?models=reverted&revids=456789|4567890
-    @bp.route("/scores/<wiki>/", methods=["GET"])
-    def score_model_revisions(wiki):
+    @bp.route("/scores/<context>/", methods=["GET"])
```

Comment:
Just for curiosity on the terminology: are there "contexts" which are not "wikis"? In other words, is a "context" a more general concept than a "wiki"?